src/static/riot/competitions/detail/leaderboards.tag

```diff
@@ -69,6 +70,31 @@
             })
         }
 
+        self.exportLeaderboardtoCSV = function () {
+            var leaderboardTitle = self.selected_leaderboard.title
```

Comment:
What if someone wants to hit our API to get this CSV? There are "CSV renderers" for DRF we should be using